DIRECTORY.md

```diff
@@ -251,4 +252,4 @@
 ## Strings
   * [Brute Force String Searching](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/strings/brute_force_string_searching.cpp)
   * [Knuth Morris Pratt](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/strings/knuth_morris_pratt.cpp)
-  * [Rabin Karp](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/strings/rabin_karp.cpp)
+  * [Rabin Karp](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/strings/rabin_karp.cpp)
```

Comment:
`DIRECTORY.md` is still being modified.
IMO, this file shouldn't be modified here.

Reply:
Yeah it was accidentally included in this PR. It looks like when I updated my fork I got commits which you guys reverted.

Forking repo again after this PR ✔️